src/Whim/Template/whim.config.csx

```diff
@@ -113,7 +113,8 @@ void DoConfig(IContext context)
 			(id) => SliceLayouts.CreatePrimaryStackLayout(context, sliceLayoutPlugin, id),
 			(id) => SliceLayouts.CreateSecondaryPrimaryLayout(context, sliceLayoutPlugin, id),
 			(id) => new FocusLayoutEngine(id),
-			(id) => new TreeLayoutEngine(context, treeLayoutPlugin, id)
+			(id) => new TreeLayoutEngine(context, treeLayoutPlugin, id),
+			(id) => new FreeLayoutEngine(context, id)
 		};
```

Comment:
It seems to me that `LayoutPreviewPlugin` is not very meaningful with the `FreeLayoutEngine`.

Reply:
I was thinking that the `LayoutPreviewPlugin` should ignore the `FreeLayoutEngine`. 

This is one of the things which would be easier if `FreeLayoutEngine` was in `Whim` as opposed to `Whim.FloatingLayout` to avoid `Whim.LayoutPreview` having another dependency. However, this is fairly arbitrary, and I wouldn't be opposed to adding `Whim.FloatingLayout` as a dependency for `Whim.LayoutPreview`.